src/vs/workbench/services/suggest/browser/media/suggest.css

```diff
@@ -62,6 +62,18 @@
 	display: flex;
 }
 
+.monaco-workbench .workbench-suggest-widget .suggest-status-bar {
```

Comment:
copied from the editor's css file

Reply:
https://github.com/microsoft/vscode/blob/478d09d2694b29da972b5038d0ce227eca3db4bd/src/vs/editor/contrib/suggest/browser/media/suggest.css#L41-L51